content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+
+GNOME Boxes has some features that require drivers to be installed into the Guest operating system for better performance and to better integrate the VM with the VM software.
+
+#### Linux Guests
+
+There are 2 drivers needed to get the full functionality out of your Linux VM with GNOME Boxes. The first is the `spice-vdagent` driver, and the other is the `spice-webdavd` driver. Both drivers are provided by the Open Source [Spice Project](https://www.spice-space.org/).
+
+Some Linux distributions include one or both of these drivers out of the box, while others will need them both installed. Additionally, your Linux distribution may not use the names used above, so you may need to reach out to those maintainers for more information. You can see an example of this discrepency below in the Arch Linux section. On Arch, the `spice-webdavd` package does not exist, but the `phodav` package works as a replacement.
+
+- Ubuntu
+
+```bash
+sudo apt update
+sudo apt install spice-webdavd
+```
+
+- Pop
+
+```bash
+sudo apt update
+sudo apt install spice-vdagent spice-webdavd
+```
+
+- Fedora
+
+```bash
+sudo dnf install spice-webdavd
+```
+
+- Arch
+
+```bash
+sudo pacman -Sy spice-vdagent phodav
+```
+
+##### VD Agent
+
+This driver brings in support for what most users will need. After installing this package and rebooting the VM, you will notice the VM will resize to match the size of your GNOME Boxes window. Additionally you will now be able to copy and paste between the Host and the VM. Both of these features are enabled by default once the driver is installed, so no configuration necessary.
+
+##### WebDAVd
+
+This driver allows you to share folders and files between the Host's Operating system, and the VM's Operating System.
+
+#### Windows Guests
+
+On Windows, there are also 2 drivers that will need installed for all the features of GNOME Boxes to be supported by Windows. These drivers are available on the [Spice Project's](https://www.spice-space.org/download.html#windows-binaries) download page. There are a number of downloads listed in this section, but the 2 drivers we need are the [Windows guest tools](https://gitlab.freedesktop.org/spice/win32/spice-nsis) and the [Spice WebDAVd](https://git.gnome.org/browse/phodav/tree/spice) drivers.
+
+##### Windows Driver Downloads
+
+- Windows Guest Tools: [https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe](https://www.spice-space.org/download/windows/spice-guest-tools/spice-guest-tools-latest.exe)
+- Spice WebDAVd: [https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x64-latest.msi](https://www.spice-space.org/download/windows/spice-webdavd/spice-webdavd-x64-latest.msi)
+
+> While trying to download the Spice WebDAVd file in the Edge browser, if I clicked the link, I was taken to a page trying to display the file as text instead of downloading the file. This ends up showing a lot of odd letters and characters.
```

Comment:
Minor formatting & clarity - It's obvious this is offered to help users troubleshoot common pitfalls, but the sudden first-person is an odd convention compared to the rest of the article. Consider adding a "Troubleshooting" header, and/or converting these first-person scenarios to standard issue-resolution format: 

Issues: 

- Clicking the WebDavd file in an Edge Browser session takes the user to page that tries to display the file as jumbled text instead of downloading the file.
- After multiple download attempts, a user sees a window with random text or receives a "This page isn't responding" prompt. 

Resolution: Right click the link above and select "Save Link As..." and save teh file to your downloads folder.